### PR TITLE
Add `base_path` setting to `atmos` config

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Add `base_path` setting to `atmos` config
* Add `atmos` logos
* Update README

## why
* Provide/configure a base path for all components and stacks
* Allow controlling stacks and components paths using only one setting or ENV var
* Can also be set using `ATMOS_BASE_PATH` ENV var, or `--base-path` command-line argument
* Supports both absolute and relative paths
* If not provided or is an empty string, `components.terraform.base_path`, `components.helmfile.base_path` and `stacks.base_path`are independent settings (supporting both absolute and relative paths)
* If `base_path` is provided (via ENV var or command-line arg), `components.terraform.base_path`, `components.helmfile.base_path` and `stacks.base_path` are considered paths relative to `base_path`

## test

```
ATMOS_BASE_PATH=./examples/complete atmos terraform apply test/test-component -s=tenant1-ue2-dev
```

```
Found ENV var ATMOS_BASE_PATH=./examples/complete

Variables for the component 'test/test-component' in the stack 'tenant1/ue2/dev':

enabled: true
environment: ue2
namespace: eg
region: us-east-2
service_1_name: service-1
service_2_name: service-2
stage: dev
tenant: tenant1

Writing variables to file:
examples/complete/components/terraform/test/test-component/tenant1-ue2-dev-test-component.terraform.tfvars.json
```

### CODE REVIEW

**Review by nitrocode — APPROVED** [review]

**Comment by nitrocode** [comment]
Is this new atmos logo in creative commons ?

**Comment by nitrocode** [comment]
Does this close PR https://github.com/cloudposse/atmos/pull/96 ?

**Comment by aknysh** [comment]
> Does this close PR #96 ?

No, it's a diff thing (providing paths to the code via inputs, not reading them from atmos.yaml)

File: README.yaml [hunk 1]
```diff
@@ -70,13 +74,13 @@ introduction: |-
   ### cloudposse/packages
   
   On Debian, run:
- 
+
```
Comment (osterman) [inline README.yaml:77]:
~~~suggestion
  Add the Cloud Posse package repository hosted by cloudsmith.
  
 ```
# Add CloudPosse package repo
apt-get update && apt-get install -y apt-utils curl
curl -1sLf 'https://dl.cloudsmith.io/public/cloudposse/packages/cfg/setup/bash.deb.sh' | bash
```
~~~

File: README.yaml [hunk 2]
```diff
@@ -70,13 +74,13 @@ introduction: |-
...
   On Debian, run:
- 
+  
   ```
   apt install atmos@="<ATMOS_VERSION>-*"
   ```
   
   On Alpine, run:
- 
+
```
Comment (osterman) [inline README.yaml:83]:
~~~suggestion
  Install the Cloud Posse package repository hosted by cloud smith.

```
curl -sSL https://apk.cloudposse.com/install.sh | bash
```
~~~

File: README.yaml [hunk 3]
```diff
@@ -24,7 +24,11 @@ categories:
   - kubernetes
   - aws
 
-description: Universal Tool for DevOps and Cloud Automation.
+description: |-
+  
+  `atmos` - Universal Tool for DevOps and Cloud Automation.
+  
+  ![atmos](docs/img/atmos-logo-128.png)
```
Comment (osterman) [inline README.yaml:31]: Use the `logo` field. 

https://github.com/cloudposse/build-harness/blob/master/README.yaml#L11

And probably best to use SVG

Comment (aknysh) [reply README.yaml:31]: SVG is big, and I'm not sure how to correctly set the image size (especially in `logo` section`

**Review by nitrocode — APPROVED** [review]

**Review by osterman — APPROVED** [review]

**Pull request merged**